### PR TITLE
[RU]GroupLe pagination shift fix

### PR DESCRIPTION
**Pull request by e-shl**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by e-shl** [comment]
What to "run"? I didn't touch him.

File: .run/GroupLeGenerator.run.xml [hunk 1]
```diff
@@ -1,11 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```
Comment (stevenyomi) [inline .run/GroupLeGenerator.run.xml:1]: You should not delete this file.

**Review by stevenyomi — CHANGES_REQUESTED** [review]

File: multisrc/overrides/grouple/selfmanga/src/SelfManga.kt [hunk 2]
```diff
@@ -12,10 +12,7 @@ class SelfManga : GroupLe("SelfManga", "https://selfmanga.live", "ru") {
     override val id: Long = 5227602742162454547
 
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
-        val url = "$baseUrl/search/advanced?offset=${70 * (page - 1)}".toHttpUrlOrNull()!!.newBuilder()
-        if (query.isNotEmpty()) {
-            url.addQueryParameter("q", query)
-        }
+        val url = = super.searchMangaRequest(page, query, filters).url.newBuilder()
```
Comment (stevenyomi) [inline multisrc/overrides/grouple/selfmanga/src/SelfManga.kt:15]:
```suggestion
        val url = super.searchMangaRequest(page, query, filters).url.newBuilder()
```
This is causing lint error.

**Review by stevenyomi — CHANGES_REQUESTED** [review]

File: multisrc/overrides/grouple/selfmanga/src/SelfManga.kt (hunk 2 above)
Comment (stevenyomi) [reply multisrc/overrides/grouple/selfmanga/src/SelfManga.kt:15]: > Have tested the modifications by compiling and running the extension through Android Studio

Please make sure you did what you claimed to have done.

**Pull request merged**